src/App.js

```diff
@@ -1,24 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Fragment } from 'react';
+import { CourtPage } from 'pages/court';
```

Comment:
just curious, any config setting is required for using absolute path import?
and if you are vscode, can you quickly open this file by `Ctrl + click`?

Reply:
https://github.com/victor77dev/badminton-scoreboard/pull/5/files#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0R14
`baseUrl` in tsconfig.json is the path for it.

> can you quickly open this file by Ctrl + click?

Yes

Reply:
i need a plugin to do global import in gatsby as well, and i cannot quick open publicly imported file lol

Nice to see your configuration